db/migrate/20201116175220_create_items.rb

```diff
@@ -0,0 +1,18 @@
+class CreateItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :items do |t|
+      t.string :name
+      t.string :type
+      t.integer :size
```

Comment:
I think this would be better as a string. its more flexible and will allow us to do sizes like "S, M, L"